cron.go

```diff
@@ -123,6 +123,9 @@ func (c *Cron) Entries() []*Entry {
 
 // Start the cron scheduler in its own go-routine.
 func (c *Cron) Start() {
+	if c.running {
+		return
+	}
```

Comment:
The method documentation says:

```Go
// Start the cron scheduler in its own go-routine.
```

But it's not what this code does, so docs would need to be updated if this change is to be merged.